students/zelichowski_michal/lesson_02_flow_control/the_number_of_zeroes.py

```diff
@@ -0,0 +1,17 @@
+"""Given N numbers: the first number in the input is N, after that N
+integers are given. Count the number of zeros among the given integers
+and print it.
+
+You need to count the number of numbers that are equal to zero,
+not the number of zero digits. """
+
+# Read an integer:
+a = int(input())
+zeroes = 0
+for x in range(1, a+1):
```

Comment:
W plikach jak `lost_card.py` w wywołaniu range zrobiłeś tak: `for x in range(1, N + 1):`
A tutaj zrobiłeś `for x in range(1, a+1):`
Oba są ok ja preferowałbym tak jak tutaj, bez dodatkowych spacji, ale nie jest to powód do wymagania poprawki.
Niemniej zdecydowałbym się na jeden styl na przyszłość. PEP8 niestety nie podaje precyzyjnego podobnego przykładu jako referencję.